From 6d8a3e46ab503ed9b0cb71bd8e26323b849ba0d1 Mon Sep 17 00:00:00 2001
From: Eugenia Emantayev <eugenia@mellanox.co.il>
Date: Mon, 13 Aug 2012 07:27:56 +0300
Subject: [PATCH] mlx4_core: backport for log_num_mtt increase

Module is not loaded in sles10sp4 - memory allocation failure

Signed-off-by: Eugenia Emantayev <eugenia@mellanox.co.il>
---
 drivers/net/mlx4/main.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/net/mlx4/main.c b/drivers/net/mlx4/main.c
index 35ed4b6..7814541 100644
--- a/drivers/net/mlx4/main.c
+++ b/drivers/net/mlx4/main.c
@@ -129,7 +129,7 @@ static struct mlx4_profile default_profile = {
 	.num_cq		= 1 << 16,
 	.num_mcg	= 1 << 13,
 	.num_mpt	= 1 << 19,
-	.num_mtt	= 1 << 24,
+	.num_mtt	= 1 << 20,
 };
 
 static int log_num_mac = 2;
-- 
1.7.8.2

